cypress.json

```diff
@@ -3,5 +3,6 @@
   "projectId": "m45k1x",
   "reporterOptions": {
     "mochaFile": "junit.xml"
-  }
+  },
+  "defaultCommandTimeout": 15000
```

Comment:
typing the public key always resulted in timeout so I increased the defaultCommandTimeout from a default of 4000. There is a option parameter on the type command to specify a different timeout value but cypress does not seem to respect that